@xen-orchestra/lite/src/components/form/FormByteSize.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <FormInputGroup>
+    <FormInput v-model="size" type="number" />
```

Comment:
Should we add a `step` lower than 1 and a `min="0"`?

Reply:
OK for `min="0"`, but are you sure a `step` < 1 would be useful?